package.json

```diff
@@ -1,23 +1,61 @@
 {
   "name": "@medv/codejar",
-  "version": "1.0.7",
+  "version": "1.1.7",
   "description": "An embeddable code editor for the browser",
   "license": "MIT",
   "repository": "antonmedv/codejar",
   "author": "Anton Medvedev <anton@medv.io>",
   "homepage": "https://medv.io/codejar/",
-  "main": "codejar.js",
+  "main": "dist/codejar.js",
+  "module": "dist/codejar.module.js",
+  "unpkg": "dist/codejar.umd.js",
+  "source": "src/codejar.ts",
+  "types": "dist/src/codejar.d.ts",
   "files": [
-    "*.ts",
-    "*.js"
+    "src",
+    "dist",
+    "cursor/dist",
+    "cursor/src",
+    "cursor/package.json",
+    "line_numbers/dist",
+    "line_numbers/src",
+    "line_numbers/package.json"
   ],
   "scripts": {
-    "start": "tsc -w",
-    "size": "minify codejar.js --sourceType module | gzip-size"
+    "build": "npm-run-all --parallel build:*",
+    "build:codejar": "microbundle --external none",
+    "build:cursor": "microbundle --cwd cursor",
+    "build:linenumbers": "microbundle --cwd line_numbers",
```

Comment:
Again imagine if I'm going to add much more functionality. Before for me, it was just adding one file. Now, this is error-prone.

Reply:
Yes, one more file need a change of build config